manticore/platforms/evm.py

```diff
@@ -2030,12 +2033,10 @@ def CALL(self, gas, address, value, in_offset, in_size, out_offset, out_size):
 
     @CALL.pos  # type: ignore
     def CALL(self, gas, address, value, in_offset, in_size, out_offset, out_size):
-        data = self.world.last_transaction.return_data
-        if data is not None:
-            data_size = len(data)
-            size = Operators.ITEBV(256, Operators.ULT(out_size, data_size), out_size, data_size)
-            self.write_buffer(out_offset, data[:size])
-
+        data = self._return_data
```

Comment:
is the same code in every CALL* maybe we should find a name for it and make it a priv8 method

Reply:
Ah, good point. `_save_old_return_data` or something?